packages/components/src/custom-gradient-picker/test/serializer.ts

```diff
@@ -12,24 +12,31 @@ import {
 describe( 'It should serialize a gradient', () => {
 	test( 'serializeGradientColor', () => {
 		expect(
-			serializeGradientColor( { type: 'rgba', value: [ 1, 2, 3, 0.5 ] } )
+			serializeGradientColor( {
+				type: 'rgba',
+				//NTS: For entire file - our tests were using numbers, but `gradientParser` (according to DefinitelyTyped) expects strings. I've updated the tests accordingly. Alternatively we could add some logic to our serializer so it accepts both and converts numbers to strings on the fly.
+				value: [ '1', '2', '3', '0.5' ],
```

Comment:
Note for this entire file: our tests were using numbers, but `gradientParser` (according to DefinitelyTyped) expects strings. I've updated the tests accordingly. Alternatively we could add some logic to our serializer so it accepts both and converts numbers to strings on the fly.

Reply:
Internal. These changes are strictly part of a unit test, and the tests themselves are written specifically internal serialization functions that don't get called by a consumer.